galleries/examples/lines_bars_and_markers/fill_between_alpha.py

```diff
@@ -1,7 +1,7 @@
 """
-==============================
-Fill Between with transparency
-==============================
+==================================
+``fill_between`` with transparency
```

Comment:
We haven't used formatting in example titles so far. I feel it's slightly better to format function names as literal rather than just have fill_between as plain text. But no strong opinion. We can also leave out the formatting if desired.